### PR TITLE
2108 - Introduction of Island Radar (EGYP)

### PR DESCRIPTION
**Pull request by jackedwards815**
New agreement between VATSIM Argentina and VATSIM UK will take affect at 2108 AIRAC cycle.

Agreement establishes a new area control position for the Mount Pleasant Control Zone:, Island Radar (ISLAND_CTR).

### CODE REVIEW

File: FIRBoundaries.dat [hunk 1]
```diff
@@ -30998,4 +31024,4 @@ YYWE|0|0|42|-60.0|75.0|-2.0|163.0|-37.810595|143.53974
 -38.000491|144.37864
 -38.009167|144.389722
 -38.143333|144.363611
--38.770556|143.263889
```
Comment (meltinglava) [inline FIRBoundaries.dat:31001]: This line should be reverted. New line at end of file should be there.

Comment (jackedwards815) [reply FIRBoundaries.dat:31001]: @meltinglava I'm not sure what's happened here as the lines appear to be the same :/ Why is it highlighting a change?

Comment (jackedwards815) [reply FIRBoundaries.dat]: Fixed.

**Review by Adrian2k — COMMENTED** [review]
Shouldn't SFAL also be changed so it belongs to EGYP?
image

**Comment by jackedwards815** [comment]
> Shouldn't SFAL also be changed so it belongs to EGYP?
> image

Done.

**Review by Adrian2k — APPROVED** [review]
That was quick 😅

**Review by NelisV — APPROVED** [review]

**Pull request merged**